lib/sirf_exercises/__init__.py

```diff
@@ -34,4 +35,22 @@ def exercises_data_path(*data_type):
             "Exercises data weren't found. Please run download_data.sh in the "
             "scripts directory")
 
-    return os.path.join(data_path, *data_type)
\ No newline at end of file
+    return os.path.join(data_path, *data_type)
+
+
+def cd_to_working_dir(*exercise_name):
+    '''
+    Creates and changes the current directory to the a working directory for the
+    current exercise, based on exercise_name.
+
+    Implementation detail: this is defined as
+    {exercises_data_path()}/working_folder/{exercise_name}. If multiple
+    exercise_name's are given, they will be treated as subdirectories.
+
+
+    exercise_name: name of the current exercise, the path will include this.
+    Multiple arguments can be given for subdirectories.
+    '''
```

Comment:
```suggestion
def cd_to_working_dir(*subfolders):
    '''
```suggestion
    Creates and changes the current directory to a working directory for the
    current exercise, based on the argument(s). If multiple
    strings are given, they will be treated as subdirectories.

    Implementation detail: this is defined as
    {exercises_data_path()}/working_folder/subfolder1/subfolder2. 

    subfolders: the path will include this.
    Multiple arguments can be given for nested subdirectories.
    '''
```
or something like this. I find it confusing to call it `exercise_name` here